src/framework/AppStore/AppStoreContext.tsx

```diff
@@ -164,6 +164,12 @@ export const AppStoreProvider = withFeatureFlags<React.PropsWithChildren<unknown
       accountId
     })
   )
+  const redirectUserToModuleHome = (featureFlagsMap: Partial<Record<FeatureFlag, boolean>>): void => {
+    if (featureFlagsMap.CREATE_DEFAULT_PROJECT && source === 'signup' && module && !isPurposePage) {
+      const moduleUrlWithDefaultProject = getModuleToDefaultURLMap(accountId, module)[module]
+      history.push(moduleUrlWithDefaultProject ? (moduleUrlWithDefaultProject as string) : routes.toHome({ accountId }))
+    }
+  }
```

Comment:
nitpick: it's worth adding some extra line breaks to make the code more scannable.
```suggestion

  const redirectUserToModuleHome = (featureFlagsMap: Partial<Record<FeatureFlag, boolean>>): void => {
    if (featureFlagsMap.CREATE_DEFAULT_PROJECT && source === 'signup' && module && !isPurposePage) {
      const moduleUrlWithDefaultProject = getModuleToDefaultURLMap(accountId, module)[module]
      history.push(moduleUrlWithDefaultProject ? (moduleUrlWithDefaultProject as string) : routes.toHome({ accountId }))
    }
  }

```